src/utils/appInstaller/index.ts

```diff
@@ -135,16 +135,25 @@ async function _installExternalApp(
     network: string,
     fullName: string,
     rootAccount: string,
+    bre: BuidlerRuntimeEnvironment,
     version?: string
   ): Promise<{ contractAddress: string; contentUri: string }> {
-    const infuraProvider = new ethers.providers.InfuraProvider(network)
+    const networkConfig: any = bre.config.networks[network]
+    const jsonRpcProvider = new ethers.providers.JsonRpcProvider(
+      networkConfig.url,
+      {
+        name: network,
```

Comment:
This needs to be 'homestead' instead of 'mainnet' since we're using ethers, right?

Reply:
I changed it to 'mainnet' because it didn't seem to work with 'homestead'.